vic/drivers/classic/src/compute_cell_area.c

```diff
@@ -0,0 +1,72 @@
+/******************************************************************************
+* @section DESCRIPTION
+*
+* This subroutine computes grid cell area.
+*
+* @section LICENSE
+*
+* The Variable Infiltration Capacity (VIC) macroscale hydrological model
+* Copyright (C) 2014 The Land Surface Hydrology Group, Department of Civil
+* and Environmental Engineering, University of Washington.
+*
+* The VIC model is free software; you can redistribute it and/or
+* modify it under the terms of the GNU General Public License
+* as published by the Free Software Foundation; either version 2
+* of the License, or (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License along with
+* this program; if not, write to the Free Software Foundation, Inc.,
+* 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+******************************************************************************/
+
+#include <vic_def.h>
+#include <vic_run.h>
+#include <vic_driver_shared.h>
```

Comment:
OK, fixed.

On Thu, Feb 11, 2016 at 5:12 PM, Joe Hamman <notifications@github.com>
wrote:

> In vic/drivers/classic/src/compute_cell_area.c
> <https://github.com/UW-Hydro/VIC/pull/388#discussion_r52694472>:
>
> > +* as published by the Free Software Foundation; either version 2
> > +* of the License, or (at your option) any later version.
> > +*
> > +* This program is distributed in the hope that it will be useful,
> > +* but WITHOUT ANY WARRANTY; without even the implied warranty of
> > +* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
> > +* GNU General Public License for more details.
> > +*
> > +* You should have received a copy of the GNU General Public License along with
> > +* this program; if not, write to the Free Software Foundation, Inc.,
> > +* 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
> > +******************************************************************************/
> > +
> > +#include <vic_def.h>
> > +#include <vic_run.h>
> > +#include <vic_driver_shared.h>
>
> <vic_driver_classic.h>?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UW-Hydro/VIC/pull/388/files#r52694472>.
>